app/controllers/posts_controller.rb

```diff
@@ -15,7 +15,6 @@ def new
   end
 
```

Comment:
- [ ] Kindly ensure to use _CanCanCan_ for Post authorization in _Post Controller_, which should be similar to:

```ruby
  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```

Reply:
@rloterh Alright let me check into that :+1: